fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -27,6 +27,9 @@
 import static javax.ws.rs.core.Response.Status.GONE;
 import static javax.ws.rs.core.Response.Status.NO_CONTENT;
 import static javax.ws.rs.core.Response.Status.OK;
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
```

Comment:
Even more minor: imports are typically sorted alphabetically, so the HttpHeaders should appear before the MediaType class.